packages/gatsby/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "@sentry/gatsby",
+  "version": "5.16.1",
+  "description": "Offical Sentry SDK for Gatsby.js",
+  "repository": "git://github.com/getsentry/sentry-javascript.git",
+  "homepage": "https://github.com/getsentry/sentry-javascript/tree/master/packages/gatsby",
+  "author": "Sentry",
+  "license": "BSD-3-Clause",
+  "engines": {
+    "node": ">=6"
+  },
```

Comment:
We should add `keywords` according to: https://www.gatsbyjs.org/contributing/submit-to-plugin-library/ so the plugin will get added to gatsby's plugin library.